### PR TITLE
Feat/fix test allow validator recovery

### PR DESCRIPTION
**Pull request by wshino**
For testing allowValidatorRecovery, 
I created the validator2Address variable, it's not installed via isModuleInstalled. 
allowValidatorRecovery needs to be installed of validator to succeed, but if the module installed function called, the validator already been registered. 
I added the mockCall for isModuleInstalled function.

### CODE REVIEW

**Comment by SoraSuegami** [comment]
Thank you for your PR!
Did you check why the failed tests returned "LinkedList_EntryAlreadyInList(0x27cc01A4676C73fe8b6d0933Ac991BfF1D77C4da)" error?

> allowValidatorRecovery needs to be installed of validator to succeed, but if the module installed function called, the validator already been registered.

I cannot understand this part very well.

**Comment by wshino** [comment]
@SoraSuegami Af far as I see, the above failed details are following steps in test_AllowValidatorRecovery_When_SafeAddOwnerSelector:

1. call setUp() in UnitBase()
2. call instance.installModule with module is validatorAddress(0x27cc01A4676C73fe8b6d0933Ac991BfF1D77C4da)
3. call onInstall function in UniversalEmailRecoveryModule
4. call allowValidatorRecovery with module is validatorAddress(0x27cc01A4676C73fe8b6d0933Ac991BfF1D77C4da)
5. call validators[0x8D778546350bCe5b88083612A87451EB0E6693b2].push(0x27cc01A4676C73fe8b6d0933Ac991BfF1D77C4da)
6. call emailRecoveryModule.allowValidatorRecovery at allowValidatorRecovery.t.sol with validatorAddress(0x27cc01A4676C73fe8b6d0933Ac991BfF1D77C4da)
7. call validators[0x8D778546350bCe5b88083612A87451EB0E6693b2].push(0x27cc01A4676C73fe8b6d0933Ac991BfF1D77C4da) -> fail

Basically onInstallModule registers validator address. So to test allowValidatorRecovery standalone, I need to add a validator without installing the module.
